docs/source/data-model.rst

```diff
@@ -20,31 +32,39 @@ JSON) in any language. Each document class has a defined, but flexible, schema.
 * Event Descriptor --- Metadata about a series of Events. Envision
   richly-detail column headings in a table, encompassing physical units,
   hardware configuration information, etc.
+* Resource --- A pointer to an external file (or resource in general).
+* Datum --- A pointer to a specific slice of data within a Resource.
 * Run Stop Document ---  Everything that we can only know at the very end, such
   as the time it ended and the exit status (succeeded, aborted, failed due to
   error).
 
+Every document has a ``time`` (its creation time) and a separate ``uid`` to
```

Comment:
This is not strictly true anymore, now that Resource and Datum are proper documents. They do not have a ``time``, and datum calls its unique id `datum_id`, an unfortunate historical choice that we are more or less stuck with. Maybe, to keep things simple, remove the mention of ``time`` here and just say, "Every document contains a unique identifier."

Reply:
This is important language.  Tripped over this before, myself.

Reply:
check